src/System.Management.Automation/utils/PowerShellExecutionHelper.cs

```diff
@@ -1,34 +1,24 @@
-
-/********************************************************************++
-Copyright (c) Microsoft Corporation.  All rights reserved.
---********************************************************************/
```

Comment:
Ok, lets rename it.

* I think `PowerShellExecutionHelper.cs` name falls into another extreme: the name is too generic and doesn't really describe what's inside the file, but naming is hard and it's a nit-pick.
* Why remove the copyright?